rpi3/firmware/optee

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
```

Comment:
Also, tee-supplicant is one thing. But I don't think the checks for the driver is necessary. Since that's something that will be loaded automatically when starting tee-supplicant.